x/attribute/keeper/genesis.go

```diff
@@ -29,7 +29,7 @@ func (k Keeper) ExportGenesis(ctx sdk.Context) (data *types.GenesisState) {
 		return nil
 	}
 
-	if err := k.IterateRecords(ctx, types.AttributeKeyPrefix, appendToRecords); err != nil {
+	if err := k.IterateRecords(ctx, types.AttributeKeyPrefixLegacy, appendToRecords); err != nil {
```

Comment:
you want to be internally consistent in using the new key prefix everywhere ... the only place the legacy one will exist is in the migration function of the module upgrade handler that transitions from version 1 (default, current) to version 2 (instance with new prefix/0.43 support)